### PR TITLE
h2 1.4.186

### PR DESCRIPTION
**Pull request by hanxue**
Remove first line that causes error
Create a symlink h2.jar that is used by other applications that need to reference H2

### CODE REVIEW

File: Library/Formula/h2.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
-require 'formula'
```
Comment (mikemcquaid) [inline Library/Formula/h2.rb]: If removing this can you fix up everything else flagged by `brew audit --strict`? Thanks

File: Library/Formula/h2.rb [hunk 2]
```diff
@@ -21,6 +19,9 @@ def install
 
     libexec.install Dir['*']
     (bin+'h2').write script
+
+    # Expose the H2 JAR as version agnostic h2.jar
+    libexec.install_symlink libexec/"bin/h2-#{version}.jar" => "h2.jar"
```
Comment (adamv) [inline Library/Formula/h2.rb]: Why is this needed?
